src/main/java/robosky/uplands/entity/HexahaenEntity.java

```diff
@@ -0,0 +1,120 @@
+package robosky.uplands.entity;
+
+import net.minecraft.entity.*;
+import net.minecraft.entity.ai.goal.*;
+import net.minecraft.entity.attribute.EntityAttributes;
+import net.minecraft.entity.data.DataTracker;
+import net.minecraft.entity.data.TrackedData;
+import net.minecraft.entity.data.TrackedDataHandlerRegistry;
+import net.minecraft.entity.mob.HostileEntity;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.world.IWorld;
+import net.minecraft.world.LocalDifficulty;
+import net.minecraft.world.World;
+import scala.Int;
+
+import javax.annotation.Nullable;
+
+public class HexahaenEntity extends HostileEntity {
+
+    private static final TrackedData<Integer> STRENGTH;
+    public int strength;
```

Comment:
Maybe just have a getter that calls the tracked data instead? So you don't have to mirror the strength across a field and tracked data.

Reply:
It was like that in the Scala version and I just adapted it. I changed it now though

Reply:
The Scala did in fact use a getter method (getters in Scala can look like properties)